### PR TITLE
Allow passing relative root  when running inCluster

### PR DESCRIPTION
**Pull request by olegsu**
related to #382

### CODE REVIEW

File: lib/config.js [hunk 1]
```diff
@@ -32,16 +32,16 @@ function defaultConfigPaths () {
 function getInCluster () {
   const host = process.env.KUBERNETES_SERVICE_HOST
   const port = process.env.KUBERNETES_SERVICE_PORT
-
+  const root = process.env.RELATIVE_ROOT || '/'
```
Comment (silasbw) [inline lib/config.js]: Can we pick a kubernetes-client specific name for this environment variable?  How about `KUBERNETES_CLIENT_SERVICEACCOUNT_ROOT`?

Comment (silasbw) [inline lib/config.js]: Could we also set the default to `/var/run/secrets/kubernetes.io/serviceaccount/` and update `caPath`, `tokenPath`, and `namespacePath` to be like `${ KUBERNETES_CLIENT_SERVICEACCOUNT_ROOT }/*`?  I'm guessing that would implement the behavior folks have in mind when they set the `KUBERNETES_CLIENT_SERVICEACCOUNT_ROOT` value.

Comment (olegsu) [reply lib/config.js]: Sounds good, will do asap

Comment (olegsu) [reply lib/config.js]: Not sure I fully understand, can you please give an example?

**Comment by olegsu** [comment]
@silasbw please check this out.
I've updated the name to be `KUBERNETES_CLIENT_SERVICEACCOUNT_ROOT`.
So it supports my example as:
```sh
export KUBERNETES_CLIENT_SERVICEACCOUNT_ROOT=$TELEPRESENCE_ROOT/var/run/secrets/kubernetes.io/serviceaccount/
```

File: lib/config.js (hunk 1 above)
Comment (silasbw) [reply lib/config.js]: The updates you made were what I had in mind. :+1:

**Comment by silasbw** [comment]
Published your change with `kubernetes-client@6.7.0`.

**Comment by olegsu** [comment]
Awesome, thanks

**Pull request merged**